packages/suite/src/views/wallet/coinmarket/common/FiatAmount.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useFormatters } from '@suite-common/formatters';
+
+interface FiatAmountProps {
+    amount?: string | number;
+    currency?: string;
+}
+
+export const FiatAmount = ({ amount, currency }: FiatAmountProps) => {
```

Comment:
Could you rename both components to something like `CoinmarketFiatAmount`?

Reply:
Done.